Readability.js

```diff
@@ -593,14 +593,25 @@ Readability.prototype = {
     }
 
     // If there's a separator in the title, first remove the final part
-    if (/ [\|\-\\\/>»] /.test(curTitle)) {
-      titleHadHierarchicalSeparators = / [\\\/>»] /.test(curTitle);
-      let allSeparators = Array.from(origTitle.matchAll(/ [\|\-\\\/>»] /gi));
+    const titleSeparators = /\|\-–—\\\/>»/;
```

Comment:
As we only ever use `.source`, we could perhaps do that here instead of requesting it repeatedly.